.github/workflows/on-merge.yml

```diff
@@ -0,0 +1,62 @@
+name: On merge
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        shell: bash
+        run: yarn install --frozen-lockfile
+      - name: Run Build
+        shell: bash
+        run: yarn build
+
+  release-publish:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+      - uses: webfactory/ssh-agent@v0.5.4
+        with:
+          ssh-private-key: |
+            ${{ secrets.PERF_BASTION_PRIVATE_KEY }}
+            ${{ secrets.JAHIA_BASTION_PRIVATE_KEY }}
+      - uses: actions/download-artifact@v3
```

Comment:
@Janin-Michel-Mathias What is this for ? You're downloading an artifact called build-artifacts, but I don't see it uploaded anywhere else on this repo.